RecoParticleFlow/PFProducer/plugins/kdtrees/KDTreeLinkerPSEcal.h

```diff
@@ -2,8 +2,8 @@
 #define KDTreeLinkerPSEcal_h
 
 #include "RecoParticleFlow/PFProducer/interface/KDTreeLinkerBase.h"
-#include "RecoParticleFlow/PFProducer/interface/KDTreeLinkerTools.h"
-#include "RecoParticleFlow/PFProducer/interface/KDTreeLinkerAlgo.h"
+#include "RecoPixelVertexing/PixelTriplets/plugins/KDTreeLinkerAlgo.h"
+#include "RecoPixelVertexing/PixelTriplets/plugins/KDTreeLinkerTools.h"
```

Comment:
cross-package includes from plugins are verboten.
I'm a bit surprised that this didn't require an update in the BuildFile.

Reply:
The `KDTreeLinker*` are header-only.